package.json

```diff
@@ -9,6 +9,7 @@
     "@mui/icons-material": "^5.6.2",
     "@mui/material": "^5.2.2",
     "@mui/styles": "^5.4.1",
+    "@mui/system": "^5.15.9",
```

Comment:
Maybe no necessary  to add ```"@mui/system": "^5.15.9"```, because you can import Box from ```@mui/material```

Reply:
![image](https://github.com/Space2Study-UA-1125/frontEnd/assets/93076661/05e045fb-63e9-4a70-8b9c-4b1461ba76e6)


yes, thanks, I changed the path